akka-http2-support/src/main/scala/akka/http/impl/engine/http2/Http2Blueprint.scala

```diff
@@ -51,6 +59,7 @@ object Http2Blueprint {
    */
   def httpLayer(): BidiFlow[HttpResponse, Http2SubStream, Http2SubStream, HttpRequest, NotUsed] = {
     val incomingRequests = Flow[Http2SubStream].via(new HttpRequestHeaderHpackDecompression)
+      .log(getClass.getSimpleName)
```

Comment:
Yes, we should make sure that errors are not swallowed but are surfaced in user level streams. Maybe add a pending test case stub for it?